docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc

```diff
@@ -1,33 +1,374 @@
+----
+Python::
++
+[source,python]
+----
+# The Data Descriptor for RangeInt64
+builder = opendaq.DataDescriptorBuilder()
+builder.sample_type = opendaq.SampleType.RangeInt64
+descriptor = builder.build()
+----
+====
+
+[#last_value_complex_number]
+== Getting the last value with a Complex Number
+
+In the following example, we first extract the last value via `getLastValue` (`last_value` in Python) call, then we cast the `lastValue` object into a `IComplexNumber` type (not needed in Python). Lastly, call the methods of the underlying type such as the `getReal` (`real` in Python) and the `getImaginary` (`imaginary` in Python), in this case provided by the `IComplexNumber` interface.
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+// Get last value of a Signal
+auto lastValue = signal.getLastValue();
+// Cast to IComplexNumber
+auto complex = lastValue.asPtr<IComplexNumber>();
+// Extract values
+auto real = complex.getReal();
+auto imaginary = complex.getImaginary();
+----
+Python::
++
+[source,python]
+----
+# Get last value of a Signal
+last_value = signal.last_value
+# Extract values
+real = last_value.real
+imaginary = last_value.imaginary
+====
+
+A Data Descriptor for a Complex Number might be:
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+// The Data Descriptor for ComplexFloat32
+auto descriptor = DataDescriptorBuilder().setSampleType(SampleType::ComplexFloat32).build();
+----
+Python::
++
+[source,python]
+----
+# The Data Descriptor for ComplexFloat32
+builder = opendaq.DataDescriptorBuilder()
+builder.sample_type = opendaq.SampleType.ComplexFloat32
+descriptor = builder.build()
+----
+====
+
+[#last_value_struct]
+== Getting the last value with a Struct
+
+A Data Descriptor for a Struct might be:
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+// Create a Data Descriptor
+auto descriptor = DataDescriptorBuilder()
+					  .setName("MyStruct")
+					  .setSampleType(SampleType::Struct)
+					  .setStructFields(List<DataDescriptorPtr>(
+						  DataDescriptorBuilder()
+							  .setName("MyInt32")
+							  .setSampleType(SampleType::Int32)
+							  .build(),
+						  DataDescriptorBuilder()
+							  .setName("MyFloat64")
+							  .setSampleType(SampleType::Float64)
+							  .build()))
+					  .build();
+// Set the Data Descriptor, thereby adding MyStruct to the Type Manager
+signal.setDescriptor(descriptor);
+----
+Python::
++
+[source,python]
+----
+# Create a Data Descriptor
+builder_int = opendaq.DataDescriptorBuilder()
+builder_int.name = "MyInt32"
+builder_int.sample_type = opendaq.SampleType.Int32
+descriptor_int = builder_int.build()
+
+builder_float = opendaq.DataDescriptorBuilder()
+builder_float.name = "MyFloat64"
+builder_float.sample_type = opendaq.SampleType.Float64
+descriptor_float = builder_float.build()
+
+builder = opendaq.DataDescriptorBuilder()
+builder.name = "MyStruct"
+builder.sample_type = opendaq.SampleType.Struct  # TODO not implemented
```

Comment:
SampleType.Struct is not supported in Python yet.

Reply:
add note in doc